test-utils/src/test_env.rs

```diff
@@ -203,7 +205,7 @@ pub async fn setup_test_programs_with_accounts() -> EnvWithAccounts {
         .process_transaction(transaction.clone())
         .await
         .unwrap();
-    create_queue_account(
+    create_indexed_array_account(
```

Comment:
what about the following?
```suggestion
    create_nullifier_queue_account(
```

Reply:
I was thinking about such a rename too, but at the end I didn't want to blow up the PR even more.